From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 5 Jun 2026 17:15:21 +0800
Subject: local-biarch

# DP: Description: Allow ldconfig to work on i386/x86-64 biarch systems
# DP: Related bugs: 
# DP: Dpatch author: Daniel Jacobowitz
# DP: Patch author: Daniel Jacobowitz
# DP: Upstream status: Debian-Specific
# DP: Status Details: based on Ubuntu change by Jeff Bailey
# DP: Date: 2005-10-13

# DP: Description: Allow ldconfig to work on i386/x86-64 biarch systems
# DP: Related bugs: 
# DP: Dpatch author: Daniel Jacobowitz
# DP: Patch author: Daniel Jacobowitz
# DP: Upstream status: Debian-Specific
# DP: Status Details: based on Ubuntu change by Jeff Bailey
# DP: Date: 2005-10-13
---
 sysdeps/unix/sysv/linux/i386/dl-cache.h | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 sysdeps/unix/sysv/linux/i386/dl-cache.h

diff --git a/sysdeps/unix/sysv/linux/i386/dl-cache.h b/sysdeps/unix/sysv/linux/i386/dl-cache.h
new file mode 100644
index 0000000..766bba4
--- /dev/null
+++ b/sysdeps/unix/sysv/linux/i386/dl-cache.h
@@ -0,0 +1 @@
+#include <sysdeps/unix/sysv/linux/sparc/dl-cache.h>
